package.json

```
@@ -137,7 +137,7 @@
     "react-native-linear-gradient": "^2.8.1",
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "1.0.98",
+    "react-native-onyx": "1.0.104",
```

Can we bump this up so we include the revert of what was causing the regression here? https://github.com/Expensify/react-native-onyx/pull/397
```suggestion
    "react-native-onyx": "1.0.105",
```